Examples/Example-Objective-C/Base.lproj/Main.storyboard

```diff
@@ -74,5 +74,13 @@
             </objects>
             <point key="canvasLocation" x="-436" y="103"/>
         </scene>
+        <!--Navigation-->
+        <scene sceneID="QgB-cn-TrA">
+            <objects>
+                <viewControllerPlaceholder storyboardName="Navigation" bundleIdentifier="org.cocoapods.MapboxNavigationUI" id="hvP-3P-Yih" sceneMemberID="viewController"/>
```

Comment:
Is this external storyboard reference still accurate, now that the MapboxNavigationUI target has been moved into a Carthage-managed project?